labelit/mainapp/templates/registration/login.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+{% load static %}
+{% block extrahead %}
+<link rel="stylesheet" href="../../static/login.css">
```

Comment:
We can use static tag instead : https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#std:templatetag-static